mesa/space.py

```diff
@@ -4,9 +4,13 @@
 
 Objects used to add a spatial component to a model.
 
-Grid: base grid, a simple list-of-lists.
-SingleGrid: grid which strictly enforces one object per cell.
-MultiGrid: extension to Grid where each cell is a set of objects.
+Grid: base grid, it creates a rectangular grid.
+SingleGrid: extension to Grid which strictly enforces one agent per cell.
+MultiGrid: extension to Grid where each cell can contain a set of agents.
+HexGrid: extension to Grid to handle hexagonal neighbors.
+ContinuousSpace: a two-dimensional space where each agent has an arbitrary
+position of `float`'s.
```

Comment:
We can improve the wording more later. Though this line needs to be indented
```
ContinuousSpace: a two-dimensional space where each agent has an arbitrary
                              position of `float`'s.
```

Reply:
done